x/txfees/keeper/genesis.go

```diff
@@ -17,12 +17,30 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
 	if err != nil {
 		panic(err)
 	}
+
+	// We track the txfees generated in the KVStore.
+	// If the values were exported, we set them here.
+	// If the values were not exported, we initialize them to zero as well as use the current block height.
+	if genState.TxFeesTracker != nil {
+		k.SetTxFeesTrackerValue(ctx, genState.TxFeesTracker.TxFees)
+		k.SetTxFeesTrackerStartHeight(ctx, genState.TxFeesTracker.HeightAccountingStartsFrom)
+	} else {
+		k.SetTxFeesTrackerValue(ctx, sdk.NewCoins())
+		k.SetTxFeesTrackerStartHeight(ctx, ctx.BlockHeight())
```

Comment:
Can we add tests for this as well?

Reply:
Yeah, sadly there were no genesis tests to start with so I am going to have to write them for the entire init/export methods, but agreed it should be done will do it now

Reply:
Genesis test added here https://github.com/osmosis-labs/osmosis/pull/6804/commits/b7447b901c1deadaca91b447e65608dd82e7d3e6